cmd/kubeadm/app/util/runtime/runtime.go

```diff
@@ -233,9 +217,5 @@ func detectCRISocketImpl(isSocket func(string) bool) (string, error) {
 
 // DetectCRISocket uses a list of known CRI sockets to detect one. If more than one or none is discovered, an error is returned.
 func DetectCRISocket() (string, error) {
```

Comment:
We may want to change the language from "socket" to "endpoint" throughout this PR

Reply:
I agree with that. The term "socket" is not true in the Windows named pipe case, however let's not do that now.
The config format uses the term "CRISocket" too, so changing that is not convenient at this point.

Reply:
this was pretty much my conclusion about the existing "socket" naming. we cannot change it at this point.